### PR TITLE
Replaced Thread.sleep() with Awaitility

### PR DESCRIPTION
**Pull request by JanaganSaravanan**
Replace has been done in one file. Change is Thread.sleep() to Awaitility.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -249,7 +249,7 @@ Checks on exceptions should always make sure to assert that a particular excepti
 
 ### Awaitility
 
-Avoid using `Thread.sleep()` in tests as it leads to long test durations and flaky behavior if a condition takes slightly longer than expected.
+Avoid using `Awaitility` in tests as it leads to long test durations and flaky behavior if a condition takes slightly longer than expected.
```
Comment (nastra) [inline CONTRIBUTING.md:252]: the original description is correct. we don't want to use `Thread.sleep()` in tests and rather want to use Awaitility

**Comment by nastra** [comment]
@JanaganSaravanan the original description is correct. we don't want to use `Thread.sleep()` in tests and rather want to use Awaitility.
Where we want to replace `Thread.sleep()` usage is in the test code itself.

**Comment by JanaganSaravanan** [comment]
Sure. Where can i find the tests file ?

**Comment by nastra** [comment]
@JanaganSaravanan there are actually many test files. Just search for `Thread.sleep()` usage across the codebase. For example, `TestIcebergSourceContinuous` is one where we'd like to replace `Thread.sleep()` usage with Awaitility

**Pull request closed without merging**